pyvera/__init__.py

```diff
@@ -20,6 +21,8 @@
 SUBSCRIPTION_MIN_WAIT = 200
 # Timeout for requests calls, as vera sometimes just sits on sockets.
 TIMEOUT = SUBSCRIPTION_WAIT
+# VeraLock set target timeout in seconds
+LOCK_TARGET_TIMEOUT = 30
```

Comment:
This seems quite long - would think 5-10s should be plenty, although worth testing.